contents/index.mdx

```diff
@@ -33,7 +33,7 @@ images:
     <Logo />
 </h1>
 
-<span class="text-base font-medium mb-0">We make dev tools that help product engineers build successful products.</span>
+<span class="text-base font-medium mb-0">We make dev tools that help developers build successful products.</span>
```

Comment:
@timgl Strong opinion weakly held, but I don't think we should remove product engineers here. It's still a thing we believe in + people do associate us with the concept. I think saying "we build dev tools" is enough to build the association between "developers" and "product engineers" as analogous.